src/app/components/pages/IsaacCompetition/CompetitionInformation/CompetitionTimeline.tsx

```diff
@@ -1,27 +1,46 @@
 import React from "react";
-import { Card, CardBody, CardText, CardTitle } from "reactstrap";
+import { Card, CardBody, CardText, CardTitle, Col, Row } from "reactstrap";
 
-interface TimelineCardProps {
+interface TimelineEntry {
+  event: string;
+  date: string;
+}
+
+interface CompetitionTimelineProps {
   title: string;
-  content: string[];
-  className?: string;
+  content: string;
+  entries: TimelineEntry[];
 }
 
-const CompetitionTimeline = ({ title, content, className = "" }: TimelineCardProps) => {
-  return (
-    <Card className={`h-100 ${className} competition-information-no-border`}>
-      <CardTitle tag="h3" className="competition-timeline-header pt-4 px-4">
-        {title}
-      </CardTitle>
-      <CardBody>
-        {content.map((text, index) => (
-          <CardText key={index} tag="h3" className="competition-timeline-date">
-            {text}
-          </CardText>
-        ))}
-      </CardBody>
-    </Card>
-  );
-};
+const CompetitionTimeline = ({ title, content, entries }: CompetitionTimelineProps) => (
+  <section className="competition-timeline">
+    <h3 className="competition-timeline-title">{title}</h3>
+    <p className="competition-timeline-content">{content}</p>
+    <Row className="p-4">
+      {entries.map((entry, index) => (
+        <Col
+          key={index}
+          xs={12}
+          sm={12}
+          md={12}
+          lg={3}
+          className={`competition-timeline-box-container competition-timeline-background-${index + 1}`}
+        >
+          <Card className={`h-100 competition-timeline-background-${index + 1} competition-information-no-border`}>
+            <CardTitle tag="h3" className="competition-timeline-header pt-4 px-4">
+              {entry.event}
+            </CardTitle>
+            <CardBody>
+              <CardText tag="h3" className="competition-timeline-date">
```

Comment:
can we just double check the padding on this text, it's not quite in line with the title on desktop: 

<img width="260" alt="image" src="https://github.com/user-attachments/assets/6507e594-ed71-47e2-8867-ed6b98bafa03">

Reply:
Resolved this by reducing the sizing from `px-4` to `px-3` under `CardTitle`

<img width="1153" alt="Screenshot 2024-11-28 at 13 15 25" src="https://github.com/user-attachments/assets/73b9b132-786b-44e1-9d4a-a2e192d807f8">